mapie_v1/integration_tests/tests/test_regression.py

```diff
@@ -195,56 +184,148 @@ def test_intervals_and_predictions_exact_equality_cross(params_cross):
-    )
-    v1 = CrossConformalRegressor(
-        **filter_params(CrossConformalRegressor.__init__, v1_params)
-    )
+    v0, v1 = initialize_models("cross", v0_params, v1_params)
+    evaluate_models(v0, v1, X, y, v0_params, v1_params)
 
-    v0_fit_params = filter_params(v0.fit, v0_params)
-    v1_fit_params = filter_params(v1.fit, v1_params)
-    v1_conformalize_params = filter_params(v1.conformalize, v1_params)
 
-    v0.fit(X_cross, y_cross, **v0_fit_params)
-    v1.fit(X_cross, y_cross, **v1_fit_params)
-    v1.conformalize(X_cross, y_cross, **v1_conformalize_params)
-
-    v0_predict_params = filter_params(v0.predict, v0_params)
-    v1_predict_params = filter_params(v1.predict, v1_params)
-    v1_predict_set_params = filter_params(v1.predict_set, v1_params)
+params_test_cases_jackknife = [
+    {
+        "v0": {
+            "alpha": 0.2,
+            "conformity_score": AbsoluteConformityScore(),
+            "cv": Subsample(n_resamplings=10, random_state=RANDOM_STATE),
+            "agg_function": "median",
+            "ensemble": True,
+            "method": "plus",
+            "sample_weight": sample_weight,
+            "random_state": RANDOM_STATE,
+        },
+        "v1": {
+            "confidence_level": 0.8,
+            "conformity_score": "absolute",
+            "resampling": 10,
+            "aggregation_method": "median",
+            "method": "plus",
+            "fit_params": {"sample_weight": sample_weight},
+            "random_state": RANDOM_STATE,
+        },
+    },
+    {
+        "v0": {
+            "estimator": positive_predictor,
+            "alpha": [0.5, 0.5],
+            "conformity_score": GammaConformityScore(),
+            "aggregation_method": "mean",
+            "cv": Subsample(n_resamplings=3,
+                            replace=True,
+                            random_state=RANDOM_STATE),
+            "method": "plus",
+            "optimize_beta": True,
+            "random_state": RANDOM_STATE,
+        },
+        "v1": {
+            "estimator": positive_predictor,
+            "confidence_level": [0.5, 0.5],
+            "aggregation_method": "mean",
+            "conformity_score": "gamma",
+            "resampling": Subsample(n_resamplings=3,
+                                    replace=True,
+                                    random_state=RANDOM_STATE),
+            "method": "plus",
+            "minimize_interval_width": True,
+            "random_state": RANDOM_STATE,
+        },
+    }
+]
 
-    v0_preds, v0_pred_intervals = v0.predict(X_cross, **v0_predict_params)
 
-    v1_pred_intervals = v1.predict_set(X_cross, **v1_predict_set_params)
-    if v1_pred_intervals.ndim == 2:
-        v1_pred_intervals = np.expand_dims(v1_pred_intervals, axis=2)
-    v1_preds = v1.predict(X_cross, **v1_predict_params)
+@pytest.mark.parametrize("params_jackknife", params_test_cases_jackknife)
+def test_jackknife_after_bootstrap_consistency(params_jackknife):
```

Comment:
We should have the same names for similar tests

Reply:
replaced by test_intervals_and_predictions_exact_equality_jackknife